README.rst

```diff
@@ -0,0 +1,56 @@
+===========
+aa-intercom
+===========
+|travis|_ |pypi|_ |coveralls|_ |requiresio|_
+
+Django integration for Intercom_ API
```

Comment:
Please describe what is it doing and how to use it. We can use it for more the user model, right?